rocAL/source/loaders/image/image_read_and_decode.cpp

```diff
@@ -279,54 +289,107 @@ ImageReadAndDecode::load(unsigned char *buff,
         for (size_t i = 0; i < _batch_size; i++)
             _decompressed_buff_ptrs[i] = buff + image_size * i;
 
+        if (_decoder_config._type != DecoderType::ROCJPEG_DEC) {
 #pragma omp parallel for num_threads(_num_threads)  // default(none) TBD: option disabled in Ubuntu 20.04
```

Comment:
we only mention Ub22+ support. Can remove this comment

Reply:
Done